lxd/config/safe.go

```diff
@@ -18,7 +18,7 @@ func SafeLoad(schema Schema, values map[string]string) (Map, error) {
 		}
 
 		for _, error := range errors {
-			message := fmt.Sprintf("Invalid configuration key: %s", error.Reason)
+			message := fmt.Sprintf("Invalid configuration key: %q", error.Reason)
```

Comment:
Is `error.Reason` just a key or message? Just asking to avoid having double double-quotes (for example: `Invalid configuration key: "key: "test""`)

Reply:
Thanks, you are right, this one will lead to unwanted nested quotes. Will fix

Reply:
fixed.